nix/docker/README.md

```diff
@@ -230,9 +230,6 @@ Note that once ledger state is converted, the cardano-node container will need
 to be run with a node configuration aligned with the new ledger state type,
 otherwise ledger replay from genesis will re-occur.
 
-For more info, see the [UTxO Migration Guide](https://ouroboros-consensus.cardano.intersectmbo.org/docs/for-developers/utxo-hd/migrating/).
```

Comment:
I've looked for the correct link. It seems to be https://ouroboros-consensus.cardano.intersectmbo.org/docs/references/miscellaneous/utxo-hd/migrating